.github/workflows/enforceTestCoverage.yml

```diff
@@ -50,7 +50,7 @@ jobs:
 
           echo "Filtering coverage report to only contain coverage info for 'src/'' folder now"
 
-          npx ts-node script/utils/filter_lcov.ts lcov.info lcov-filtered.info 'test/' 'script/'
+          npx tsx script/utils/filter_lcov.ts lcov.info lcov-filtered.info 'test/' 'script/'
```

Comment:
I don't think you need to use npx and tsx. They both do the exact same thing. Should just be `tsx script/...`

Reply:
I tried it without before and it didnt work, even though tsx is part of our dependencies in package.json

It works with `yarn tsx ...` and `npx tsx ...` but not without 

![image](https://github.com/user-attachments/assets/54d2a155-4c77-471e-a85b-9dfe564c31fe)

Is there a problem when we use it this way?